include/mega/types.h

```diff
@@ -865,6 +865,7 @@ class SyncConfig : public Cacheable
         TYPE_UP = 0x01, // sync up from local to remote
         TYPE_DOWN = 0x02, // sync down from remote to local
         TYPE_TWOWAY = TYPE_UP | TYPE_DOWN, // Two-way sync
+        TYPE_BACKUP, // special sync up from local to remote, automatically disabled when remote changed
```

Comment:
Should be explicitly indicate the value?
```suggestion
        TYPE_BACKUP = 0x04, // special sync up from local to remote, automatically disabled when remote changed
```

Reply:
I don't think so. It is previous value +1, and future-proof, in case previous values change.